### PR TITLE
Reserving colons

### LINKED ISSUE #28: Contribute to more than the default namespace?
SDF 1.1 models contribute their definitions to the default namespace.
Should there be a way to contribute to other namespaces as well?
How can this be done without violating the RFC 6901 JSON pointer fragment identifier syntax?

### PR DESCRIPTION
**Pull request by cabo**
Close #28

### CODE REVIEW

File: sdf.md [hunk 1]
```diff
@@ -485,14 +485,80 @@ complex devices that will use one or more `sdfObject` definitions.
...
+For any particular JSON object in an SDF model, the set of member
+names that are used is either of:
+
+* A set of "*Quality Names*", where the entries in the object are
+  Qualities.  Quality Names are defined by the present specification
+  and its extensions, together with specific semantics to be
+  associated with the member value given with a certain Quality Name.
+
+* A set of "*Given Names*", where the entries in the object are separate
+  entities (definitions, declarations, etc.) that each are freely named by
```
Comment (akeranen) [inline sdf.md:509]: "freely named" sounds quite liberal. I wonder if just "named" is better?

File: sdf.md [hunk 2]
```diff
@@ -485,14 +485,80 @@ complex devices that will use one or more `sdfObject` definitions.
...
+  the SDF model author so these names can be employed by a user of that model.
+
+In a path from the root of the tree to any leaf, Quality Names and
+Given Names roughly alternate (with the information block,
+{{information-block}}, as a prominent exception).
+
+The meaning of the JSON object that is the member value associated
+with a Given Name is derived from the Quality Name that was used as
+the member name associated to parent.
```
Comment (akeranen) [inline sdf.md:518]: "a parent"?

**Review by akeranen — CHANGES_REQUESTED** [review]
A few small comments/questions but otherwise LGTM

File: sdf.md [hunk 3]
```diff
@@ -485,14 +485,80 @@ complex devices that will use one or more `sdfObject` definitions.
...
+
+For any particular JSON object in an SDF model, the set of member
+names that are used is either of:
+
+* A set of "*Quality Names*", where the entries in the object are
+  Qualities.  Quality Names are defined by the present specification
+  and its extensions, together with specific semantics to be
+  associated with the member value given with a certain Quality Name.
+
+* A set of "*Given Names*", where the entries in the object are separate
```
[the remark this answers is not in the document]
Comment (cabo) [reply sdf.md:508]: Namespace names aren't exactly definition names, and I was including them in the concept of "given names".

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md:509]: The point is that the model creator is free to choose the name, so "freely" was trying to emphasize that.   What is wrong about being liberal here?

File: sdf.md (hunk 2 above)
Comment (cabo) [reply sdf.md:518]:
```suggestion
the member name associated to the parent.
```

actually.

File: sdf.md (hunk 1 above)
Comment (cabo) [reply sdf.md:509]: "without constraint"?

**Pull request merged**